src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -533,6 +534,11 @@ public void afterSave(@NonNull SCMNavigatorOwner owner) {
         }
     }
 
+    @VisibleForTesting
+    GitHubWebHook getHook() {
```

Comment:
unclear why you need to refactor this method. In any case you can probably just use `@Restricted(NoExternalUse.class)` rather than `@VisibleForTesting` even if you think you need this method